manifests/server/config.pp

```diff
@@ -119,7 +119,7 @@
     } else {
       # Template has available user-supplied data
       # - $config_data
-      $cfg_content = template('mongodb/mongodb.conf.2.6.erb')
+      $cfg_content = template('mongodb/mongodb.conf.erb')
```

Comment:
Isn't the name here reflecting the fact that the template produces 2.6 formatted config?
Wouldn't it be better to keep the filename and introduce yaml based config in a new template?

Reply:
That sounds reasonable.